tablo/models.py

```diff
@@ -542,30 +581,30 @@ def get_quantile_breaks(self, field, break_count):
 
     def get_natural_breaks(self, field, break_count):
 
-        if not self._validate_fields([field]):
-            raise ValueError('Invalid field')
-
-        sql_statement = """SELECT service_table.{field} FROM
-                            (SELECT ROW_NUMBER() OVER (ORDER BY {field}) AS row_number,
```

Comment:
We really don't have a standard for SQL formatting, and I think it needs to be treated a bit differently than code because of the aliasing. Sorry for opening up the can of worms, but you brought it on yourself by editing this code outside of scope. :) 

I would suggest the following modification:
```
            SELECT service_table.{field}
            FROM {table} service_table,
                (
                    SELECT ROW_NUMBER() OVER (ORDER BY {field}) AS row_number, {primary_key}
                    FROM {table}
                    WHERE {field} IS NOT NULL
                    ORDER BY {field}
                ) all_data,
                (
                    SELECT 
                        COUNT(0) AS total,
                        CASE
                            WHEN COUNT(0) < {num_samples} THEN 1
                            ELSE ROUND((COUNT(0) / {num_samples}), 0)
                        END AS how_many
                    FROM {table}
                    WHERE {field} IS NOT NULL
                ) count_table
            WHERE service_table.{primary_key} = all_data.{primary_key} AND
                (
                    MOD(row_number, how_many) = 0 OR
                    row_number IN (1, total)
                )
            ORDER BY service_table.{field}
```

This keeps the main clause sections of the query lined up on the left side and easy to break out but still keeps the indent space consistent. I also found myself being confused by the alias between parens, as in `) all_data, (`, so I moved that next paren to the next line. Also, the indent on the case makes it clear where it starts and ends.

I would say similar changes to the quantile breaks.

For testing, this code is executed by styling a point dataset, choosing class breaks and choosing natural breaks. Changing the number of breaks will cause the code to be executed quickly in succession.